core/baseline-core-v1.0-psd01.md

```diff
@@ -2903,27 +2954,43 @@ BPI Edge Storage MUST ensure eventual consistency between edge storage and BPI u
 * *All messages will eventually reach their intended recipients* 
 * *After a certain, yet unknown, time the network will become synchronous again*
 
+[[R303]](#r303) Testability: Eventual consistency between different BPI storage can be achieved utilizing a P2P messaging solution with eventual consistency for data synchronization. Such a P2P solution is [libp2p](https://github.com/libp2p/go-libp2p) with its complete test suite. Therefore, the requirement is testable 
+
 #### **[R304]**	
 Replication conflicts MUST be automatically detectable. 
 
+[[R304]](#r304) Testability: An example of external BPI storage that can detect replication conflicts automatically is the open-source [PostgreSQL](https://www.postgresql.org/) data base, and since PostgreSQL has an extensive test suite including replication tests, the requirement is testable.
+
 #### **[R305]** 
 Replication conflicts MUST be resolvable either automatically or manually.
 
+[[R305]](#r305) Testability: An example of external BPI storage that can resolve replication conflicts automatically (e.g. rejecting conflicing updates), as well as manually (e.g. update a table update SQL procedure) is the open-source [PostgreSQL](https://www.postgresql.org/) data base, and since PostgreSQL has an extensive test suite including replication tests, the requirement is testable.
```

Comment:
```suggestion
[[R305]](#r305) Testability: An example of external BPI storage that can resolve replication conflicts automatically (e.g. rejecting conflicting updates), as well as manually (e.g. update a table update SQL procedure) is the open-source [PostgreSQL](https://www.postgresql.org/) data base, and since PostgreSQL has an extensive test suite including replication tests, the requirement is testable.
```